### ISSUE TITLE
subject column in mysql need be tinyblob

### ISSUE THREAD

**Issue by lord-alfred** [flashmob/go-guerrilla#187]
Need to change this column type in [wiki](https://github.com/flashmob/go-guerrilla/wiki/Configuration-example:-save-to-Redis-&-MySQL) for new users.
With varchar type was getting errors like:
```
time="2019-08-30T03:39:13+03:00" level=error msg="There was a problem the insert" error="Error 1366: Incorrect string value: '\\xCC\\xE5\\xF4\\xE1\\xF7\\xE5...' for column 'subject' at row 1"
time="2019-08-30T03:39:13+03:00" level=error msg="doQuery error" error="Error 1366: Incorrect string value: '\\xCC\\xE5\\xF4\\xE1\\xF7\\xE5...' for column 'subject' at row 1"
time="2019-08-30T03:39:13+03:00" level=error msg="storage error"
```

After change `subject` column type to `tinyblob`:
``subject` tinyblob NOT NULL,`

the error is gone :)

`tinyblob` has 255 symbols length, as is previous varchar: https://dev.mysql.com/doc/refman/8.0/en/string-type-overview.html

**Comment by flashmob** [comment]
Thanks for reporting this.
This happens even after the column's charset was changed to utf8mb4?
The only reason to choose VARCHAR was so that a query for the column using
= / LIKE would be case-insensitive.
As mentioned before, if MySQL is in strict mode off, you won't see these
errors...

**Comment by lord-alfred** [comment]
Yes, there were errors even after changing the encoding to `utf8mb4`

**Comment by flashmob** [comment]
Strange. How about changing the connection charset, as mentioned by the
following article?
https://hackernoon.com/today-i-learned-storing-emoji-to-mysql-with-golang-204a093454b7

**Comment by lord-alfred** [comment]
Connection string already as is article and database of course use `CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci` too
